frontend/src/pages/SignupPage/index.module.css

```diff
@@ -1,15 +1,12 @@
-.main {
+.component {
```

Comment:
Aiii ... making me very nervous this naming. So many components everywhere all day long.